src/Zttp.php

```diff
@@ -54,6 +54,11 @@ function asFormParams()
         return $this->bodyFormat('form_params')->contentType('application/x-www-form-urlencoded');
     }
 
+    function asMultipart()
+    {
+        return $this->bodyFormat('multipart')->contentType("multipart/form-data; boundary=".md5(uniqid(null, true)));
```

Comment:
Is the `boundary` part (or even the content type actually, I realize I did that on the others but maybe not needed?) totally necessary? Guzzle won't take care of that with just the body format set?